python/ray/tune/logger.py

```diff
@@ -58,17 +63,35 @@ def flush(self):
 class UnifiedLogger(Logger):
     """Unified result logger for TensorBoard, rllab/viskit, plain json.
 
-    This class also periodically syncs output to the given upload uri."""
+    This class also periodically syncs output to the given upload uri.
+
+    Arguments:
+        config: Configuration passed to all logger creators.
+        logdir: Directory for all logger creators to log to.
+        upload_uri (str): Optional URI where the logdir is sync'ed to.
+        custom_loggers (list): List of custom logger creators.
+        sync_cmd_tmpl (str): Optional template for syncer to run. Needs to
+            include replacement fields "{local_dir}" and "{remote_dir}".
+            See ray/python/ray/tune/log_sync.py
```

Comment:
Hey @hartikainen, can you leave some feedback in #3034? The sentiment there is similar, but it would be great to get some consolidated feedback before implementing.